pkg/vulnerabilityreport/scanner.go

```diff
@@ -115,12 +116,25 @@ func (s *Scanner) getCredentials(ctx context.Context, spec corev1.PodSpec, ns st
 	return kube.MapContainerNamesToDockerAuths(kube.GetContainerImagesFromPodSpec(spec), imagePullSecrets)
 }
 
-func (s *Scanner) prepareScanJob(workload kube.Object, spec corev1.PodSpec, credentials map[string]docker.Auth) (*batchv1.Job, []*corev1.Secret, error) {
+func getVulnerabilityScannerPodTolerations(starboardConfigData starboard.ConfigData) []corev1.Toleration {
```

Comment:
In the future we'll have to add the same code for other types of scanners. Can we add *GetScanJobTolerations()* or a similar method to the *ConfigData* type and use it here and potentially other places?

Reply:
For sure.

Reply:
This is done.